### PR TITLE
Move generation to babel 7

### PR DESCRIPTION
**Pull request by DanielMSchmidt**
This allows us to have nicer comments in our generated code (and we need to do it anyways at some point).

### CODE REVIEW

File: detox/src/devices/IosDriver.js [hunk 1]
```diff
@@ -3,9 +3,9 @@ const fs = require('fs');
 const DeviceDriverBase = require('./DeviceDriverBase');
 const InvocationManager = require('../invoke').InvocationManager;
 const invoke = require('../invoke');
-const GREYConfigurationApi = require('./../ios/earlgreyapi/GREYConfiguration');
-const GREYConfigurationDetox = require('./../ios/earlgreyapi/GREYConfigurationDetox');
-const EarlyGrey = require('./../ios/earlgreyapi/EarlGrey');
+const GREYConfigurationApi = require('./../ios/earlgreyapi/GREYConfiguration'); // eslint-disable-line node/no-unpublished-require
```
Comment (rotemmiz) [inline detox/src/devices/IosDriver.js]: Please try to understand why this eslint doesn't like those lines. It makes no sense.

Comment (DanielMSchmidt) [reply detox/src/devices/IosDriver.js]: I can remove it, should be red on master too, right 🤷‍♂️ The explanation with the npmignore makes kinda sense though

**Comment by rotemmiz** [comment]
@DanielMSchmidt, can we go forward with this PR? Can you please rerun it on the updated code?

**Comment by DanielMSchmidt** [comment]
Sure, it was just out of focus previously 👍  Will put it on my list to rerun the generation again.

**Comment by rotemmiz** [comment]
So is this ready?

**Pull request merged**